### PR TITLE
cryptobattles

### PR DESCRIPTION
**Pull request by Ksuyaresko**

### CODE REVIEW

File: config/cryptobattles.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "preprocessor": "cryptobattles.js", // the file name of the preprocessor script (see the preprocessor example below)
```
Comment (byyyyyyyyy) [inline config/cryptobattles.json]: comments aren't allowed on a json file. kindly remove them.

File: preprocessor/cryptobattles.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = (nftdata) => ({ 
+    name: nftdata.name,
```
Comment (byyyyyyyyy) [inline preprocessor/cryptobattles.js:2]: any additional properties that you want to show? image, etc.

**Review by byyyyyyyyy — COMMENTED** [review]
additional change requests

File: preprocessor/cryptobattles.js (hunk 2 above)
Comment (Ksuyaresko) [reply preprocessor/cryptobattles.js:2]: There is no metadata in our nft

File: config/cryptobattles.json (hunk 1 above)
Comment (Ksuyaresko) [reply config/cryptobattles.json]: done

File: config/cryptobattles.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        "BSC": "0x1034E332aEe10a0d0e0017c607B1276aCA541022"
+      },
+      "production": {
+        "BSC": ""
+      }
+    }
+  },
+  "abi": {
+    "filename": "abi-nft.json",
+    "source": "https://github.com/Ksuyaresko/betting-abi/blob/master/abi-nft.json"
```
Comment (ed3ath) [inline config/cryptobattles.json]: source must be in raw if it's hosted in github

File: config/cryptobattles.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+  "preprocessor": "cryptobattles.js",
+  "template": "",
+  "type": "other-example",
+  "artsgenerator": "",
+  "token": {
+    "method": "get",
+    "contractAddress": { 
+      "test": {
+        "BSC": "0x1034E332aEe10a0d0e0017c607B1276aCA541022"
```
Comment (ed3ath) [inline config/cryptobattles.json]: please provide a valid token address.

Comment (byyyyyyyyy) [inline config/cryptobattles.json]: address is not existing on testnet

File: config/cryptobattles.json (hunk 3 above)
Comment (byyyyyyyyy) [inline config/cryptobattles.json]: need exactly to return the json data

File: config/cryptobattles.json (hunk 4 above)
Comment (Ksuyaresko) [reply config/cryptobattles.json]: this is test nft on binance smart chain, not testnet

File: config/cryptobattles.json (hunk 3 above)
Comment (Ksuyaresko) [reply config/cryptobattles.json]: sorry, I don't understand that exactly you mean. It is link to .json file. What t should be?

File: config/cryptobattles.json (hunk 4 above)
Comment (ed3ath) [reply config/cryptobattles.json]: please provide a nft address for testnet, not mainnet.

File: config/cryptobattles.json (hunk 3 above)
Comment (ed3ath) [reply config/cryptobattles.json]: you provided a link to github page. the link must return a json response.

Comment (byyyyyyyyy) [reply config/cryptobattles.json]: sample abi for our Character nft. you can try visiting. https://app.cryptoblades.io/abi/Characters.json

File: config/cryptobattles.json (hunk 4 above)
Comment (byyyyyyyyy) [reply config/cryptobattles.json]: hmm i see. the test object is expecting testnet contract addresses.

**Review by ed3ath — CHANGES_REQUESTED** [review]

File: config/cryptobattles.json (hunk 4 above)
Comment (Ksuyaresko) [reply config/cryptobattles.json]: binance testnet ? or rinkeby is ok?

File: config/cryptobattles.json (hunk 3 above)
Comment (Ksuyaresko) [reply config/cryptobattles.json]: done

File: config/cryptobattles.json (hunk 4 above)
Comment (Ksuyaresko) [reply config/cryptobattles.json]: done

File: config/cryptobattles.json (hunk 3 above)
Comment (Ksuyaresko) [reply config/cryptobattles.json]: done

File: config/cryptobattles.json (hunk 4 above)
Comment (byyyyyyyyy) [reply config/cryptobattles.json]: we don't have support for rinkeby. we do have for bsc though

Comment (Ksuyaresko) [reply config/cryptobattles.json]: deployed to BSC testnet

**Comment by ed3ath** [comment]
@Ksuyaresko can you mint at least 5 nfts on bsc tesnet and sent them to this address? 0x2548696795a3bCd6A8fAe7602fc26DD95A612574

**Comment by Ksuyaresko** [comment]
> @Ksuyaresko can you mint at least 5 nfts on bsc tesnet and sent them to this address? 0x2548696795a3bCd6A8fAe7602fc26DD95A612574

done. tokenId: 1, 2, 3, 4, 5, 6

**Pull request merged**